### PR TITLE
ENH add back glibc override

### PR DESCRIPTION
**Pull request by beckermr**
Let's try this again.

### CODE REVIEW

File: conda_forge_tick/mamba_solver.py [hunk 1]
```diff
@@ -242,6 +242,8 @@ def is_recipe_solvable(feedstock_dir) -> Tuple[bool, List[str], Dict[str, bool]]
         A lookup by variant config that shows if a particular config is solvable
     """
 
+    os.environ['CONDA_OVERRIDE_GLIBC'] = '2.50'
```
Comment (beckermr) [inline conda_forge_tick/mamba_solver.py:245]:
```suggestion
    os.environ["CONDA_OVERRIDE_GLIBC"] = "2.50"
```

File: conda_forge_tick/mamba_solver.py [hunk 2]
```diff
@@ -288,6 +290,8 @@ def is_recipe_solvable(feedstock_dir) -> Tuple[bool, List[str], Dict[str, bool]]
         errors.extend([f"{cbc_name}: {e}" for e in _errors])
         solvable_by_cbc[cbc_name] = _solvable
 
+    del os.environ['CONDA_OVERRIDE_GLIBC']
```
Comment (beckermr) [inline conda_forge_tick/mamba_solver.py:293]:
```suggestion
    del os.environ["CONDA_OVERRIDE_GLIBC"]
```

**Comment by isuruf** [comment]
When was this removed?

**Comment by beckermr** [comment]
In a previous PR. I am hacking on the cuda migration trying to get the jobs unstuck. This code didn't help and so I tried something else. Now I am trying both things.

**Pull request merged**